src/mrack/utils.py

```diff
@@ -165,17 +166,25 @@ def is_windows_host(meta_host):
 
     Host is windows host if:
     * os starts with 'win' or
-    * os_type is 'windows'
+    * os_type is 'windows' or
+    * group starts with 'WIN' or
+    * group uppercase starts with 'AD'
     """
     return (
         meta_host.get("os", "").startswith("win")
         or meta_host.get("os_type", "") == "windows"
+        or meta_host.get("group", "").upper().startswith("WIN")
+        or meta_host.get("group", "").upper().startswith("AD")
     )
 
 
 def get_username(host, meta_host, config):
     """Find username from sources db/metadata/config."""
-    username = host.username or meta_host.get("username")
+    if host:
```

Comment:
In what situation is this change needed? It seems to me that in all usages the host is defined.

Reply:
This was probably one time error i will remove it